python/railjson_generator/railjson_generator/scripts/examples/small_infra.py

```diff
@@ -627,13 +627,16 @@ def add_signal_on_ports(switch, ports: Mapping[str, Tuple[str, str]]):
     stops=[Stop(location=loc, duration=60) for loc in stop_locations],
 )
 
+train_0.add_standard_single_value_allowance("percent", 5)
```

Comment:
Small_infra will have everything except `AllowanceTimeValue`. Replace one of the 2 `AllowancePercentValue` to test the time value, or add it to another train.